xcm/xcm_token_locations.json

```diff
@@ -243,22 +463,218 @@
         "interiors": {
           "v1": [
             {
-              "Parachain": 2090
+              "parachain": 2090
             },
             {
-              "GeneralIndex": 0
+              "generalIndex": 0
             }
           ],
           "v3": [
             {
-              "Parachain": 2090
+              "parachain": 2090
+            },
+            {
+              "generalIndex": 0
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "RMRK",
```

Comment:
`RMRK` is a token in Kusama ecosystem. It must have `v3` version for `interiors` block

Reply:
Fixed